polish-adblock-filters/adblock_ublock.txt

```diff
@@ -433,24 +431,24 @@ bezprawnik.pl##.wall-bg[style^="background-image:"]:style(background-image: none
 cda-tv.pl##+js(nano-setTimeout-booster.js, TheLink)
 !
 ! wprost.pl
-wprost.pl##div[class*="lazyload"]:style(background: none !important;)
 wprost.pl##div[class*="lazyload"] > a > img:style(display: inline-block !important; height: 100% !important; width: auto !important;)
+wprost.pl##div[class*="lazyload"]:style(background: none !important;)
 !
 !
 ! Realizacje CMC Media
-ddb24.pl,iotwock.info,ototorun.pl,twojradom.pl,szczecinek.com,zycie.pila.pl,kurierpodlaski.pl,zambrow.org,egarwolin.pl,lowicz24.eu,ciechanowinaczej.pl,itvpiaseczno.pl,ikampinos.pl,infosiedlce.pl,ibialoleka.pl,izoliborz.pl,imokotow.pl,nolesnica.pl,lipno-cli.pl,golub-cgd.pl,ibielsk.pl,brodnica-cbr.pl,egorzow.pl,ipragapoludnie.pl,rypin-cry.pl,wabrzezno-cwa.pl,wio.waw.pl,e-wyszogrod.pl##.c-content__adv:style(padding-top: 0 !important;)
+brodnica-cbr.pl,ciechanowinaczej.pl,ddb24.pl,e-wyszogrod.pl,egarwolin.pl,egorzow.pl,golub-cgd.pl,ibialoleka.pl,ibielsk.pl,ikampinos.pl,imokotow.pl,infosiedlce.pl,iotwock.info,ipragapoludnie.pl,itvpiaseczno.pl,izoliborz.pl,kurierpodlaski.pl,lipno-cli.pl,lowicz24.eu,nolesnica.pl,ototorun.pl,rypin-cry.pl,szczecinek.com,twojradom.pl,wabrzezno-cwa.pl,wio.waw.pl,zambrow.org,zycie.pila.pl##.c-content__adv:style(padding-top: 0 !important;)
 !
 ! Polsat
-www.polsatsport.pl,www.polsatnews.pl##.ads .videoPlayer__vast,  .ads .videoPlayer__mask[data-time]
-||redirector.redefine.pl/r/*$media,redirect=noopmp4-1s,domain=www.polsatsport.pl|www.polsatnews.pl
+www.polsatnews.pl,www.polsatsport.pl##.ads .videoPlayer__vast, .ads .videoPlayer__mask[data-time]
+||redirector.redefine.pl/r/*$media,redirect=noop-1s.mp4,domain=www.polsatnews.pl|www.polsatsport.pl
 !
 ! it-manuals.info
 it-manuals.info###pum-822
-it-manuals.info##html.pum-open:style(overflow: auto !important;)
 it-manuals.info##html.pum-open [aria-hidden]:style(padding-right: 0 !important;)
+it-manuals.info##html.pum-open:style(overflow: auto !important;)
 !
 ! google.com,google.pl
-google.com,google.pl##.rllt__link:has(> div[class]:has-text(Reklama)):xpath(..)
+google.com,google.pl##.rllt__link:has( > div[class]:has-text(Reklama)):xpath(..)
```

Comment:
a tu po co spacja?

Reply:
On wstawia spację między B a > a C, bo tak raczej powinno być. 
Na kursie też wstawiają spacę.
`http://www.kurshtml.edu.pl/css/selektor_dziecka,elementy.html`

Reply:
to może lepiej go nauczyć by od razu kopiował tam `.rllt__link`, `:scope` niby AdGuard wywala jako niebezpieczne / niestabline / powolne.

```
google.com,google.pl##.rllt__link:has(.rllt__link > div[class]:has-text(Reklama)):xpath(..)
```

Reply:
@krystian3w Ja jestem początkujący w sprawach Pythona, ale z tego co mi wiadomo Mistrz @krystian3w da sobie ta z tym radę :smile:

Reply:
Wywala, bo pewnie dlatego, że 2 hasy i do tego jeszcze xpath, co za dużo to nie zdrowo :smile: